sidebars.js

```diff
@@ -195,6 +195,7 @@ module.exports = {
                 "components/modeler/bpmn/timer-events/timer-events",
                 "components/modeler/bpmn/error-events/error-events",
                 "components/modeler/bpmn/terminate-events/terminate-events",
+                "components/modeler/bpmn/escalation-events/escalation-events",
```

Comment:
:thought_balloon:  Same as above :smile: should we put `Escalation events` before `Terminate events` so that we maintain the same order as in the BPMN coverage events table?